crawler/config/defaults.py

```diff
@@ -45,6 +45,7 @@
             "AP_sanger_report_200726_0001.csv",
             "AP_sanger_report_200727_0001.csv",
             "AP_sanger_report_200728_0001.csv",
+            "AP_sanger_report_200729_0001.csv",
         ],
```

Comment:
Is this intentional?

Reply:
Via slack: yes, that was the change from yesterday
14:48
we changed it inside the container, but with this we update in the code, so if someone deploys we dont lose the change